arc.pl

```diff
@@ -809,32 +808,33 @@ sub make_ephin_goes_table {
     my %val;
     my %tab_def = %{$opt{ephin_goes_table}};
 
-    my $start = qr/P1 \s+ P2  \s+ P5 \s+ P8  \s+ P10 \s+ P11 \s+ H2/x;
-
-    my ($goes_date, $p2, $p5);
```

Comment:
I figured there was really no reason to *parse* the MTA proton page at this point.